setup.py

```diff
@@ -21,19 +21,19 @@ def read_version(package):
 version = read_version('connexion')
 
 install_requires = [
-    'clickclick>=1.2',
-    'jsonschema>=2.5.1',
-    'PyYAML>=5.1',
-    'requests>=2.9.1',
-    'inflection>=0.3.1',
-    'openapi-spec-validator>=0.2.4',
+    'clickclick>=1.2,<21',
+    'jsonschema>=2.5.1,<4',
+    'PyYAML>=5.1,<6',
+    'requests>=2.9.1,<3',
+    'inflection>=0.3.1,<1',
```

Comment:
Given that it's still on 0.x , breaking changes could still occur within the same major version, so perhaps we should be a bit more strict for this one and specify `<0.4` ?

Reply:
I've updated the upper limits for 0.x versions to the minor version above the current latest version. In this case, inflection is already on 0.5.1, so I used 0.6.